Tests/web3swiftTests/localTests/ST20AndSecurityTokenTests.swift

```diff
@@ -0,0 +1,141 @@
+        XCTAssertEqual(st20token.name, expectedName)
+        XCTAssertEqual(st20token.decimals, expectedDecimals)
+    }
+
+    func testST20TokenBalanceAndAllowance() async throws {
+        let expectedAllowance = BigUInt.randomInteger(lessThan: BigUInt(10000000000))
+        let expectedBalance = BigUInt.randomInteger(lessThan: BigUInt(10000000000))
+
+        let userAddress = EthereumAddress("0xe22b8979739D724343bd002F9f432F5990879901")!
+        let delegate = EthereumAddress("0x2dD33957C90880bE4Ee9fd5F703110BDA2E579EC")!
+
+        ethMock.onCallTransaction = { transaction in
+            guard let function = self.st20token.contract.contract.getFunctionCalled(transaction.data) else {
+                XCTFail("Failed to decode function call to determine what shall be returned")
+                return Data()
+            }
+            switch function.name {
+            case "balanceOf":
+                let address = function.decodeInputData(transaction.data)?["0"] as? EthereumAddress
+                XCTAssertEqual(address, userAddress)
+                return ABIEncoder.encode(types: [.uint(bits: 256)], values: [expectedBalance] as [AnyObject])!
+            case "allowance":
+                let transactionInput = function.decodeInputData(transaction.data)
+                XCTAssertEqual(transactionInput?["0"] as? EthereumAddress, userAddress)
+                XCTAssertEqual(transactionInput?["1"] as? EthereumAddress, delegate)
+                return ABIEncoder.encode(types: [.uint(bits: 256)], values: [expectedAllowance] as [AnyObject])!
+            default:
+                // Unexpected function called
+                XCTFail("Called function '\(String(describing: function.name))' which wasn't supposed to be called.")
+                return Data()
+            }
+        }
+
+        let balance = try await st20token.getBalance(account: userAddress)
+        let allowance = try await st20token.getAllowance(originalOwner: userAddress, delegate: delegate)
+        XCTAssertEqual(balance, expectedBalance)
+        XCTAssertEqual(allowance, expectedAllowance)
+    }
+
+    func testSecurityTokenInvestors() async throws {
+        let expectedNumberOfInvestors = BigUInt.randomInteger(lessThan: BigUInt(10000000000))
+//        let expectedInvestors = [EthereumAddress("0xe22b8979739D724343bd002F9f432F5990879901")!,EthereumAddress("0x2dD33957C90880bE4Ee9fd5F703110BDA2E579EC")!]
+
+        ethMock.onCallTransaction = { transaction in
+            guard let function = self.securityToken.contract.contract.getFunctionCalled(transaction.data) else {
+                XCTFail("Failed to decode function call to determine what shall be returned")
+                return Data()
+            }
+            switch function.name {
+            case "investorCount":
+                return ABIEncoder.encode(types: [.uint(bits: 256)], values: [expectedNumberOfInvestors] as [AnyObject])!
+//            case "investors":
```

Comment:
Calling `try await securityToken.investors()` crashes.
The ABI and Swift implementations do not match. In ABI it expects uint256 argument as I recall but in Swift we try to call just `investors` function without arguments.

TODO: must fix the implementation.

Reply:
Todo or Fixme? I mean do you planning to fix it within this PR or is it a task for some better times?

Reply:
Not with this PR. I'll push a separate one as it's unrelated issue.

Reply:
Removed commented code.
Fixing will be done in another PR.